### ISSUE TITLE
ESP32 Audio Kit support?

### ISSUE THREAD

**Issue by mircose** [schreibfaul1/ESP32-audioI2S#416]
Hi,
Is this library compatible with ESP32 Audio Kit?
https://docs.ai-thinker.com/en/esp32-audio-kit

**Comment by schreibfaul1** [comment]
yes you can use that, there is an older version with AC101 and a newer one with ES8388.
There is an example here: https://github.com/schreibfaul1/ESP32-audioI2S/tree/master/examples/ESP32_A1S
